profiler/shakesapp/shakesapp_main.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Google LLC
```

Comment:
Filename can probably just be main.go, given this is in the shakesapp directory?

Reply:
Done.